pkg/metricprovider/external_metric_provider.go

```diff
@@ -149,48 +171,13 @@ func (p *ExternalMetricProvider) GetCronExternalMetrics(ctx context.Context, nam
 	if len(errs) > 0 {
 		return nil, fmt.Errorf("%v", errs)
 	}
-	replicas := DefaultCronTargetMetricValue
```

Comment:
why remove this part?

Reply:
DefaultCronTargetMetricValue set to  1
change to minScales